app/views/subjects/show.json.jbuilder

```diff
@@ -1,3 +1,4 @@
 # frozen_string_literal: true
 
 json.partial! "subjects/subject", subject: @subject
+# json.array! @subjects, partial: "subjects/subject", as: :subject
```

Comment:
Do you think you'll want these again? It's good to delete code rather than commenting it out so you don't wonder what it was later.